native/swift/Tests/wordpress-api/Support/HTTPStubs.swift

```diff
@@ -1,15 +1,17 @@
 import Foundation
 import WordPressAPI
 
-#if canImport(WordPressAPIInternal)
-import WordPressAPIInternal
-#endif
+final class HTTPStubs: SafeRequestExecutor {
 
-class HTTPStubs: SafeRequestExecutor {
+    typealias Stub = (condition: @Sendable (WpNetworkRequest) -> Bool, response: WpNetworkResponse)
 
-    var stubs: [(condition: (WpNetworkRequest) -> Bool, response: WpNetworkResponse)] = []
+    private let stubs: [Stub]
+    private let missingStub: Result<WpNetworkResponse, Error>?
 
-    var missingStub: Result<WpNetworkResponse, Error>?
+    init(stubs: [Stub] = [], missingStub: Result<WpNetworkResponse, Error>? = nil) {
```

Comment:
Because `SafeRequestExecutor` needs to be `Sendable`, we need our stubs to be too.

The easiest way to do that is to just make them immutable, which doesn't change the code all that much